settings.gradle

```diff
@@ -37,4 +37,5 @@ include 'eventmesh-protocol-plugin'
 include 'eventmesh-protocol-plugin:eventmesh-protocol-api'
 include 'eventmesh-protocol-plugin:eventmesh-protocol-openmessage'
 include 'eventmesh-protocol-plugin:eventmesh-protocol-cloudevents'
+include 'eventmesh-protocol-plugin:eventmesh-protocol-grpc'
```

Comment:
In my view, the grpc is used to replace `http` and `tcp`, it can help us easy to maintain rpc code and other language sdk. It can be placed in runtime module, if I am wrong, please correct me.

Reply:
yes, maybe it needs a "eventmesh-transport-plugin" which contains eventmesh-transport-grpc/eventmesh-transport-http/eventmesh-transport-tcp etc.

Reply:
this module only contains grpc protobuf and the generated java codes. the actual implementation will be in runtime module and SDK module.

For now grpc is in experimental development. Once it is mature and in good shape, it can replace TCP/HTTP transport in the runtime module, and SDK side. Thanks.